env/env_test.go

```diff
@@ -31,7 +31,10 @@ func TestGetEnvExistingVariable(t *testing.T) {
 	const envVariableValue = "foobar"
 
 	// set the environment variable
-	os.Setenv(envVariableName, envVariableValue)
+	err := os.Setenv(envVariableName, envVariableValue)
+	if err != nil {
```

Comment:
Maybe better to extract it to separate function?

Reply:
ok it will be better, thank you